src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -5,7 +5,20 @@
  * Do not remove no-args constructor
  */
 public class Excavator extends Machine {
+    private String excavatorType;
+    private int numOfAdditionalExcavatorEquipment;
```

Comment:
Could you please also rename this field so that make it much smaller and concise